### PR TITLE
Microsalt send all samples fix

### PR DESCRIPTION
**Pull request by Mropat**
## Description
*This PR adds/fixes ...*
Fix issue when samples were sent to microsalt analysis if they had too few reads

### How to test
- [ ] create microsalt config for a case with negative controls

### Expected test outcome
- [ ] check that all samples included
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/workflow/microsalt/base.py [hunk 1]
```diff
@@ -98,11 +98,7 @@ def config_case(
         LOG.error("No sample found for that ticket/sample_id")
         raise click.Abort
 
-    parameters: List[dict] = [
-        analysis_api.get_parameters(sample_obj)
-        for sample_obj in sample_objs
-        if sample_obj.sequencing_qc
-    ]
+    parameters: List[dict] = [analysis_api.get_parameters(sample_obj) for sample_obj in sample_objs]
```
Comment (moonso) [inline cg/cli/workflow/microsalt/base.py:101]: `dict`, is it possible to expose what it is?

File: cg/meta/workflow/microsalt.py [hunk 2]
```diff
@@ -172,6 +172,7 @@ def get_parameters(self, sample_obj: models.Sample) -> Dict[str, str]:
             "date_libprep": str(sample_obj.prepared_at or datetime.min),
             "method_libprep": method_library_prep or "Not in LIMS",
             "method_sequencing": method_sequencing or "Not in LIMS",
+            "sequencing_qc_passed": sample_obj.sequencing_qc,
```
Comment (moonso) [inline cg/meta/workflow/microsalt.py:175]: Typical model stuff utilising `from_orm` in pydantic 😄

**Review by moonso — APPROVED** [review]
👍

File: cg/cli/workflow/microsalt/base.py (hunk 1 above)
Comment (Mropat) [reply cg/cli/workflow/microsalt/base.py:101]: [str, Any] isnt too helpful

**Pull request merged**